app/mailers/article_mailer.rb

```diff
@@ -9,13 +9,13 @@
 class ArticleMailer < MandrillMailer::TemplateMailer
   include ActionView::Helpers::UrlHelper
 
-  default from: 'orientation@codeschool.com'
+  default from: ENV['DEFAULT_FROM_EMAIL'] || 'orientation@codeschool.com'
```

Comment:
Actually, you remind me that this should definitely not default to a @codeschool.com email at all. Can you change this to `notifications@orientation.io`?